<!DOCTYPE HTML>
<!--
	Solid State by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Bibliography - CThe Telomere-to-Telomere Revolution: Illuminating Oomycete Genomes</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">

		<!-- Page Wrapper -->
			<div id="page-wrapper">
				<!-- Banner -->
					<section id="banner">
						<div class="inner">
							<h2>Bibliography - The Telomere-to-Telomere Revolution: Illuminating Oomycete Genomes</h2>
							<p><a href="mailto:kfletcher@ucdavis.edu" target="_blank">Kyle Fletcher</a> and Richard Michelmore, <a href="https://genomecenter.ucdavis.edu/" target="_blank">Genome Center, UC Davis</a></p>
							<p>2023 IS-MPMI CONGRESS</p>
							<ul class="contact">
								<li class="icon brands fa-twitter"><a href="https://twitter.com/KIGFletcher" target="_blank"> @kigfletcher</a></li>
								<li class="icon solid fa-envelope"><a href="mailto:kfletcher@ucdavis.edu">kfletcher@ucdavis.edu</a></li>
							</ul>
						</div>
					</section>

				<!-- Wrapper -->
					<section id="wrapper">

						<!-- One -->
							<section id="one" class="wrapper spotlight style1">
								<div class="inner">
									<a class="image"><img src="images/PeffusaFig.png" alt="" /></a>
									<div class="content">
										<h3>Ancestral Chromosomes for Family Peronosporaceae Inferred from a Telomere-to-Telomere Genome Assembly of <i>Peronospora effusa</i></h3>
										<p><font size = "3"><b>Abstract:</b> Downy mildew disease of spinach, caused by the oomycete <i>Peronospora effusa</i>, causes major losses to spinach production. In this study, the 17 chromosomes of <i>P. effusa</i> were assembled telomere-to-telomere, using Pacific Biosciences high-fidelity reads. Of these, 16 chromosomes are complete and gapless; chromosome 15 contains one gap bridging the nucleolus organizer region. This is the first telomere-to-telomere genome assembly for an oomycete. Putative centromeric regions were identified on all chromosomes. This new assembly enables a reevaluation of the genomic composition of <i>Peronospora</i> spp.; the assembly was almost double the size and contained more repeat sequences than previously reported for any <i>Peronospora</i> species. Genome fragments consistently underrepresented in six previously reported assemblies of <i>P. effusa</i> typically encoded repeats. Some genes annotated as encoding effectors were organized into multigene clusters on several chromosomes. Putative effectors were annotated on 16 of the 17 chromosomes. The intergenic distances between annotated genes were consistent with compartmentalization of the genome into gene-dense and gene-sparse regions. Genes encoding putative effectors were enriched in gene-sparse regions. The near-gapless assembly revealed apparent horizontal gene transfer from Ascomycete fungi. Gene order was highly conserved between <i>P. effusa</i> and the genetically oriented assembly of the oomycete <i>Bremia lactucae</i>; high levels of synteny were also detected with <i>Phytophthora sojae</i>. Extensive synteny between phylogenetically distant species suggests that many other oomycete species may have similar chromosome organization. Therefore, this assembly provides the foundation for genomic analyses of diverse oomycetes.</font></p>
										<a href="https://apsjournals.apsnet.org/doi/10.1094/MPMI-09-21-0227-R" target="_blank" class="special">Read the paper</a>
										<a href="https://www.ncbi.nlm.nih.gov/assembly/GCA_021491655.1" target="_blank" class="special">Download the genome (NCBI)</a>
									</div>
								</div>
							</section>

						<!-- Two -->
							<section id="two" class="wrapper alt spotlight style2">
								<div class="inner">
									<a class="image"><img src="images/PsorghiFig.png" alt="" /></a>
									<div class="content">
										<h2>The genome of the oomycete <i>Peronosclerospora sorghi</i>, a cosmopolitan pathogen of maize and sorghum, is inflated with dispersed pseudogenes</h2>
										<p><font size = "3"><b>Abstract:</b> Several species in the oomycete genus <i>Peronosclerospora</i> cause downy mildew on maize and can result in significant yield losses in Asia. Biosurveillance of these pathogens is a high priority to prevent epidemics on maize in the US and consequent damage to the US economy. The unresolved taxonomy and dearth of molecular resources for <i>Peronosclerospora</i> spp. hinder these efforts. <i>P. sorghi</i> is a pathogen of sorghum and maize with a global distribution, for which limited diversity has been detected in the southern USA. We characterized the genome, transcriptome, and mitogenome of an isolate, representing the US pathotype 6. The highly homozygous genome was assembled using 10x Genomics linked reads and scaffolded using Hi-C into 13 chromosomes. The total assembled length was 319.6 Mbâ€”larger than any other oomycete previously assembled. The mitogenome was 38 kb, similar in size to other oomycetes, although it had a unique gene order. Nearly 20,000 genes were annotated in the nuclear genome, more than described for other downy mildew causing oomycetes. The 13 chromosomes of <i>P. sorghi</i> were highly syntenic with the 17 chromosomes of <i>Peronospora effusa</i> with conserved centromeric regions and distinct chromosomal fusions. The increased assembly size and gene count of <i>P. sorghi</i> is due to extensive retrotransposition, resulting in putative pseudogenization. Ancestral genes had higher transcript abundance and were enriched for differential expression. This study provides foundational resources for analysis of <i>Peronosclerospora</i> and comparisons to other oomycete genera. Further genomic studies of global <i>Peronosclerospora</i> spp. will determine the suitability of the mitogenome, ancestral genes, and putative pseudogenes for marker development and taxonomic relationships.</font></p>
										<a href="https://www.biorxiv.org/content/10.1101/2022.07.13.499355v1" target="_blank" class="special">Read the preprint</a>
										<a href="https://www.ncbi.nlm.nih.gov/assembly/GCA_026184515.1/" target="_blank" class="special">Download the genome (NCBI)</a>
									</div>
								</div>
							</section>

						<!-- Three -->
							<section id="three" class="wrapper spotlight style3">
								<div class="inner">
									<a class="image"><img src="images/BlactucaeFig.png" alt="" /></a>
									<div class="content">
										<h2 class="major">AFLAP: assembly-free linkage analysis pipeline using k-mers from genome sequencing data</h2>
										<p><font size = "3">Our assembly-free linkage analysis pipeline (AFLAP) identifies segregating markers as k-mers in the raw reads without using a reference genome assembly for calling variants and provides genotype tables for the construction of unbiased, high-density genetic maps without a genome assembly. AFLAP is validated and contrasted to a conventional workflow using simulated data. AFLAP is applied to whole genome sequencing and genotype-by-sequencing data of F1, F2, and recombinant inbred populations of two different plant species, producing genetic maps that are concordant with genome assemblies. The AFLAP-based genetic map for <i>Bremia lactucae</i> enables the production of a chromosome-scale genome assembly.</font></p>
										<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02326-x" target="_blank" class="special">Read the paper</a>
										<a href="https://www.nature.com/articles/s41467-019-10550-0" target="_blank" class="special">Read the first <i>B. lactucae</i> genome paper</a>
										<a href="https://www.ncbi.nlm.nih.gov/assembly/GCA_004359215.2" target="_blank" class="special">Download the genome (NCBI)</a>
									</div>
								</div>
							</section>



		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
